l10n_ch_base_bank/models/bank.py

```diff
@@ -315,51 +347,41 @@ def onchange_acc_number_set_swiss_bank(self):
             self.ccp = ccp
 
     @api.onchange('ccp')
-    def onchange_ccp_set_empty_acc_number(self):
-        """ If acc_number is empty and bank ccp is defined fill it """
+    def onchange_ccp_set_acc_number(self):
+        """If ccp changing and it's a postal bank update acc_number to ccp
+        we don't want make acc_number as compute to have possibility set it
+        manually and also avoid to shadow other logic on acc_number if exist
+        """
         if self.bank_id:
-            if not self.acc_number and self.ccp:
```

Comment:
Here we are in an onchange.

We cannot assume the field is set nor is empty.
This check was there to avoid an overwrite of what the user would have written in `acc_number` field.

I think it would be annoying for the user to enter the acc_number field, then edit the ccp field and see his first edition overwritten.

Reply:
ok, than we would have obsolete data here https://github.com/i-vyshnevska/l10n-switzerland/blob/b38e620a876b8577eac326450a5af294daa8376c/l10n_ch_base_bank/tests/test_bank.py#L250 
we would have 'Camptocamp/CCP 46-110-7', ccp part wouldn't be changed
I think we should move some logic to write and give user not so complicated interface, maybe set domain on bank_id etc